File: utility_scripts/make_large_csv.py

printer = "vnr;heiti;magn"
for i in range(1, 1000000):
    if i%100 == 0:
        print(f"Still going, i: {i}")
    printer += f"i{i};product name;569\n"

with open("large.csv", 'w') as f:
    f.write(printer)

File: utility_scripts/make_large_csv.rs
use std::io::Write;

fn main() {
    let mut i = 0;
    let mut printer = String::from("vnr;heiti;magn\n");
    loop {
        if i%100 == 0 {
            println!("Still going i: {}", i);
        }
        if i > 1_000_000 {
            break;
        }
        printer.push_str(&format!("i{};product name;569\n", i));
        i+= 1;
    }
    let mut file = std::fs::File::create("large.csv").unwrap();
    file.write_all(printer.as_bytes());
}